source/numerics.tex

```diff
@@ -9671,11 +9671,11 @@
   float tgammaf(float x);
   long double tgammal(long double x);
 
-  float ceil(float x);                  // see \ref{library.c}
-  double ceil(double x);
-  long double ceil(long double x);      // see \ref{library.c}
-  float ceilf(float x);
-  long double ceill(long double x);
+  constexpr float ceil(float x);                // see \ref{library.c}
+  constexpr double ceil(double x);
+  constexpr long double ceil(long double x);    // see \ref{library.c}
+  constexpr float ceilf(float x);
+  constexpr long double ceill(long double x);
 
   float floor(float x);                 // see \ref{library.c}
   double floor(double x);
```

Comment:
Missing `constexpr` for these and the others 3 below them.
```suggestion
  constexpr float floor(float x);                 // see \ref{library.c}
  constexpr double floor(double x);
```

Reply:
fixed